edgraph/server.go

```diff
@@ -149,11 +149,11 @@ func GetGQLSchema(namespace uint64) (uid, graphQLSchema string, err error) {
 		&api.Request{
 			Query: `
 			query {
-			  ExistingGQLSchema(func: has(dgraph.graphql.schema)) {
-				uid
-				dgraph.graphql.schema
-			  }
-			}`})
+				ExistingGQLSchema(func: has(dgraph.graphql.schema)) {
+					uid
+					dgraph.graphql.schema
+				  }
+				}`})
```

Comment:
I don't see any code change happening here, do you want to roll this back? I guess your IDE settings may be doing this. We will have a larger PR to fix these aesthetics - I don't think these changes should go in as part of this.

Reply:
This is the golang fmt. It is run automatically by the VsCode. I prefer not to fight it. After all, it follows the standards recommended by the Golang maintainers.

Reply:
PS. Let me know your call.